deploy/crds/operator.ibm.com_platformapis_crd.yaml

```diff
@@ -27,8 +27,102 @@ spec:
             submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
           type: string
         spec:
-          type: object
-          x-kubernetes-preserve-unknown-fields: true
+          properties:
+            auditService:
+              type: object
+              properties:
+                config:
+                  type: object
+                  properties:
+                    auditEnabled:
+                      type: boolean
+                    auditLogPath:
+                      type: string
+                    enabled:
+                      type: boolean
+                    logrotate:
+                      type: string #really not sure if this is string or some kind of object
```

Comment:
You are correct using "string" type. https://swagger.io/docs/specification/data-models/data-types/